packages/nextjs/app/builders/page.tsx

```diff
@@ -1,41 +1,32 @@
 "use client";
 
+// Work in Progress
 import { useEffect, useState } from "react";
-import { BuilderPicture, MentorPicture } from "./_components/index";
+import Image from "next/image";
+import { ShootingStars } from "./0x5D56b71abE6cA1Dc208Ed85926178f9758fa879c/_components/shooting-stars";
+import { StarsBackground } from "./0x5D56b71abE6cA1Dc208Ed85926178f9758fa879c/_components/stars-background";
+import BuilderCard from "./_components/BuilderCard";
 import type { NextPage } from "next";
-import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+import { useScaffoldEventHistory, useScaffoldReadContract } from "~~/hooks/scaffold-eth";
 import { Builder, Mentor } from "~~/types/builders";
 
-const getRandomGender = () => {
-  return Math.random() < 0.5 ? "men" : "women";
-};
-
-const getRandomUserImage = () => {
-  const randomInt = Math.floor(Math.random() * 50);
-  const randomGender = getRandomGender();
-
-  return `https://randomuser.me/api/portraits/${randomGender}/${randomInt}.jpg`;
-};
-
 const getBuilderFromEvent = (event: any): Builder => ({
-  image: getRandomUserImage(),
   profileLink: "builders/" + event.args.builder,
   address: event.args.builder,
   checkedIn: true,
 });
```

Comment:
As a helpful tip, always make sure to check your browser console. 
Here you'll find the error:
![Screenshot 2024-09-26 at 19 28 15](https://github.com/user-attachments/assets/60b5f36a-106b-44ba-ae9f-08f0399988a6)

Reply:
i am not getting this error on localhost. why do you think it might be happening?

Reply:
so i realized that this error also is not happening on chrome console but appears on firefox ( i have no clue why )... and the weirdest part is, it is happening when ENS name & image is being fetched (even in the Address component of wagmi which is supposed to work properly).. so im thinking it is some kind of bug where we get a CORS error in the intermediate step but are able to get through eventually cause the ENS image is being fetched ultimately anyway

Reply:
Yeah really weird cannot figure out why we are getting this error. My hope is that it is only a localhost error and that it disappears when we deploy it to vercel, let's see.